browser.jsx

```diff
@@ -1,9 +1,9 @@
 'use strict'
-var remote = require('remote')
-var Menu = remote.require('menu')
-var MenuItem = remote.require('menu-item')
-var clipboard = require('clipboard')
-var urllib = require('url')
+var remote = require('electron').remote
+var Menu = require('electron')
```

Comment:
@slothbag : Thanks for your reply.I am trying to debug this.If I found the root cause will update this thread :) .